modules/ocf_printhost/files/enforcer

```diff
@@ -117,6 +117,16 @@ NOTIFY_JOB_ERROR = dedent("""\
         Please contact a staff member for assistance.\
 """)
 
+NOTIFY_PROHIBITED_FILE_TYPE = dedent("""\
+        I'm afraid I can't let you do that.
+        Please download your file as a PDF before downloading.
+        {file_type} files disagree with my software.\
```

Comment:
I'd reword this to something like, "Unfortunately we don't support printing from Google Docs. Please download your file as PDF, then try again."

Reply:
Fair enough.